neqo-transport/src/lib.rs

```diff
@@ -250,3 +253,6 @@ impl From<CloseError> for CloseReason {
 }
 
 pub type Res<T> = Result<T, Error>;
+
+// Defined in -recovery 6.2 as 333ms but using lower value.
+pub const DEFAULT_INITIAL_RTT: Duration = Duration::from_millis(100);
```

Comment:
Why move this out of `rtt.rs`?

I think I have a slight preference for defining this in `rtt.rs` as before and exposing it here as `pub use rtt::DEFAULT_INITIAL_RTT` at the top of the file.

Reply:
Good idea, fixed